lib/eosknowledgeprivate/ekn-media-bin.c

```diff
@@ -1796,17 +1823,19 @@ static void
 ekn_media_bin_init_playbin (EknMediaBin *self)
 {
   EknMediaBinPrivate *priv = EMB_PRIVATE (self);
+  GstElement *play;
```

Comment:
nitpick: Why create the local variable when you could assign directly to `priv->play`?